tests/FiberTraces.py

```diff
@@ -854,10 +794,12 @@ def suite():
 
     suites = []
     suites += unittest.makeSuite(FiberTraceTestCase)
-#    suites += unittest.makeSuite(tests.MemoryTestCase)
     return unittest.TestSuite(suites)
 
 def run(exit = False):
+    """Quiet down loggers which are too verbose"""
+    lsst.log.setLevel("CameraMapper", lsst.log.FATAL)
+    lsst.log.setLevel("afw.image.ExposureInfo", lsst.log.FATAL)
```

Comment:
I'm a little suspicious of this one.   What are the unwanted messages, and did you file a ticket upstream to get rid of them?

In general we should not be setting log levels for other packages.